src/main/java/org/apache/maven/plugins/changes/ChangesReport.java

```diff
@@ -75,6 +75,21 @@ public class ChangesReport extends AbstractChangesReport {
     @Parameter(property = "changes.addActionDate", defaultValue = "false")
     private boolean addActionDate;
 
+    /**
+     * Whether the changelog action text should be passed as raw text to Doxia for rendering the output or not.
+     * By default, the text is passed as escaped text to Doxia to support any output format like HTML, PDF, etc.
+     * If you want to produce only HTML output, and want to use HTML markup in the changelog (e.g. anchor links)
+     * set this parameter to false.
```

Comment:
This is too specific. It has to be generic:
Whether the changelog action text should be passed as raw text to the Doxia sink or not.

Reply:
updated javadoc